package.json

```diff
@@ -21,11 +21,11 @@
     "index.js"
   ],
   "scripts": {
-    "setup": "yarn install && yarn allow-scripts",
+    "setup": "yarn install",
```

Comment:
Nit: we could remove this script altogether. We left it in controllers when we migrated, but since then we've been deleting it.

Reply:
done [here](https://github.com/MetaMask/eth-simple-keyring/pull/119/commits/71ab402fa900f3ee90445b5f6a072d69aed7caa2)